src/Components/Contacts/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { memo } from "react";
+import ContactItem from "./ContactItem";
+
+const Contacts = ({contacts}) => {
```

Comment:
If your object contains only one line please set spaces after opening curly bracket "{" and before  closing curly bracket "}"